test/integration/apiserver/apiserver_test.go

```diff
@@ -1319,6 +1319,226 @@ func TestAPICRDProtobuf(t *testing.T) {
+				if err != nil {
+					t.Fatalf("unable to create cr: %v", err)
+				}
+				return cr, subresourcesCrdGVR.Group, "foosubs"
+			},
+			subresource: "scale",
+		},
+		{
+			name:   "verify status subresource returns a table for replicationcontrollers",
+			accept: "application/json;as=Table;g=meta.k8s.io;v=v1",
+			object: func(t *testing.T) (metav1.Object, string, string) {
+				rc := &v1.ReplicationController{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "replicationcontroller-1",
+					},
+					Spec: v1.ReplicationControllerSpec{
+						Replicas: int32Ptr(2),
+						Selector: map[string]string{
+							"label": "test-label",
+						},
+						Template: &v1.PodTemplateSpec{
+							ObjectMeta: metav1.ObjectMeta{
+								Labels: map[string]string{
+									"label": "test-label",
+								},
+							},
+							Spec: v1.PodSpec{
+								Containers: []v1.Container{
+									{Name: "test-name", Image: "nonexistant-image"},
+								},
+							},
+						},
+					},
+				}
+				rc, err := clientset.CoreV1().ReplicationControllers(testNamespace).Create(context.TODO(), rc, metav1.CreateOptions{})
+				if err != nil {
+					t.Fatalf("unable to create replicationcontroller: %v", err)
+				}
+				return rc, "", "replicationcontrollers"
+			},
+			subresource: "status",
+		},
+		{
+			name:   "verify scale subresource returns a table for replicationcontrollers",
+			accept: "application/json;as=Table;g=meta.k8s.io;v=v1",
+			object: func(t *testing.T) (metav1.Object, string, string) {
+				rc := &v1.ReplicationController{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "replicationcontroller-2",
+					},
+					Spec: v1.ReplicationControllerSpec{
+						Replicas: int32Ptr(2),
+						Selector: map[string]string{
+							"label": "test-label",
+						},
+						Template: &v1.PodTemplateSpec{
+							ObjectMeta: metav1.ObjectMeta{
+								Labels: map[string]string{
+									"label": "test-label",
+								},
+							},
+							Spec: v1.PodSpec{
+								Containers: []v1.Container{
+									{Name: "test-name", Image: "nonexistant-image"},
+								},
+							},
+						},
+					},
+				}
+				rc, err := clientset.CoreV1().ReplicationControllers(testNamespace).Create(context.TODO(), rc, metav1.CreateOptions{})
+				if err != nil {
+					t.Fatalf("unable to create replicationcontroller: %v", err)
+				}
+				return rc, "", "replicationcontrollers"
+			},
+			subresource: "scale",
+		},
+	}
+
+	for i := range testcases {
+		tc := testcases[i]
+		t.Run(tc.name, func(t *testing.T) {
+			obj, group, resource := tc.object(t)
+
+			cfg := dynamic.ConfigFor(config)
+			if len(group) == 0 {
+				cfg = dynamic.ConfigFor(&restclient.Config{Host: s.URL})
+				cfg.APIPath = "/api"
+			} else {
+				cfg.APIPath = "/apis"
+			}
+			cfg.GroupVersion = &schema.GroupVersion{Group: group, Version: "v1"}
+
+			client, err := restclient.RESTClientFor(cfg)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			res := client.Get().
```

Comment:
Does tabular output work for other verbs, e.g. post/put/patch (apply)? If so we should test those verbs also.

Reply:
These tests are similar to `TestTransform` that tests that the underlying transformation works as expected and uses Get as a way to trigger the underlying transformation. Do we still need to test other verbs as they all use the same underlying transform mechanisms?

Reply:
`TestTransform`: https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/apiserver_test.go#L1372

Reply:
@lavalamp @smarterclayton could you please take a look at the above review and validate if testing just `Get()` would suffice here? Thanks!

Reply:
OK